java-rmi/test/com/baeldung/rmi/JavaRMITest.java

```diff
@@ -0,0 +1,44 @@
+package com.baeldung.rmi;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.rmi.NotBoundException;
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+
+import org.junit.Test;
+
+public class JavaRMITest {
+	
+	@Test
+	public void WhenRunServer_ServerIsStarted() {
+		
+		try {
+			MessengerServiceImpl server = new MessengerServiceImpl();
+			server.createStubAndbind();
+			assertTrue(true);
+		} catch (RemoteException e) {
+			fail("Exception Occured");
```

Comment:
(No action necessary) Is this typical of unit tests you have written? Usually, I will simply put RemoteException in the throws clause.

Reply:
Either way works. This way just gives idea where it has failed.

Reply:
Thank you for adding the "then". The "when" needs to be lowercased as well. Sorry for not being clearer.